### DISCUSSION TITLE
Night scene

### DISCUSSION THREAD

**Discussion by phlnil** [blakeblackshear/frigate#12624] [General]
Hi, 
in short : even with _improve_contrast: true_ Frigate is struggling too much during the night,  increasing the fps just during the night could be a way ? (just to be clear the area has a good spot light and tracking slow object is possible... of course not so simple for faster one).
thank you

**Comment by NickM-27** [comment]
if you are referring to objects, then it is most likely not a motion detection issue but rather an object detection issue. The default models are not trained on IR images so a model trained on these images will perform much better

**Reply by phlnil** [reply]
but in my case even with IR off... the cam is quite good to catch motion and I can see image not in greyscale IR look.
the event record in fact are really rare but no zero.

**Reply by NickM-27** [reply]
Right, the statement still stands that the default models object detection models don't have examples from security cameras at all. This includes IR as well as night color. I doubt this is an issue with motion detection.

**Reply by phlnil** [reply]
Thank you NickM-27, 
so it will be feasible only using a $$ plan for + version to mitigate my case ? (tuning the model is needed ?) 
Is not relative to my HW or configuration file/options that can I modify to try to improve this issue ?

**Reply by NickM-27** [reply]
Not necessarily a paid model, just a model trained on images like the ones you are trying to detect in. 

Frigate+ is a paid service that gives you access to a model like that.
